docs/api-reference/next/image.md

```diff
@@ -71,6 +72,7 @@ The layout behavior of the image as the viewport changes size.
 | `fixed`               | Sized to `width` and `height` exactly                    | `1x`, `2x` (based on [imageSizes](#image-sizes))                                                            | N/A     |
 | `responsive`          | Scale to fit width of container                          | `640w`, `750w`, ... `2048w`, `3840w` (based on [imageSizes](#image-sizes) and [deviceSizes](#device-sizes)) | `100vw` |
 | `fill`                | Grow in both X and Y axes to fill container              | `640w`, `750w`, ... `2048w`, `3840w` (based on [imageSizes](#image-sizes) and [deviceSizes](#device-sizes)) | `100vw` |
+| `raw`                 | Insert the image element with no responsive behavior     | `640w`, `750w`, ... `2048w`, `3840w` (based on [imageSizes](#image-sizes) and [deviceSizes](#device-sizes)) | `100vw` |
```

Comment:
I changed this to "no automatic responsive behavior" which I think is accurate without implying that it cannot be made responsive. This column of the chart is describing behavior, rather than implementation, so I wouldn't be inclined to mention wrappers (or the lack thereof) at this point. Wrappers and styling are mentioned in the `raw` mode description just below this chart.

Reply:
I still think it's a bit confusing.

Even with `raw` layout, I will still have an `srcset` and the browser will still choose the right version to download from that information. To me, I feel this is an `automatic responsive behavior` from the browser.

It might just be me, but I wouldn't underestimate what `responsive` can mean in different context.

With that said, I would tend to go more with something that mention NextJs layouts. Maybe something like `Insert the image element with no automatic layout`